src/Reflection/ReflectionClass.php

```diff
@@ -438,30 +431,28 @@ public function hasMethod(string $methodName) : bool
     }
 
     /**
+     * Get an array of the defined constants for this specific class (i.e. do not search
+     * up parent classes etc.)
+     *
+     * @return mixed[]
```

Comment:
Constants can be `string[]|int[]|float[]|array[]|bool[]|null[]`. Can you remove `mixed[]` and replace it, while we're here?